### ISSUE TITLE
about us page is not added

### ISSUE THREAD

**Issue by Ritu-2001** [codervivek5/Formfil.in#6]

**Comment by Beingfarazkhan** [comment]
I would like to Contribute Towards this issue 🚀

**Comment by Beingfarazkhan** [comment]
I would like to Contribute Towards this issue Sir🌟

**Comment by manya0210** [comment]
Hello sir ..I want to contribute to this issue under jwoc

**Comment by ManasaKaza** [comment]
I want to work on this issue under JWOC

**Comment by Harry629** [comment]
I want to work on this issue under JWOC

**Comment by manya0210** [comment]
Hello sir ...I want to work on this project under jwoc

**Comment by SubhajitB05** [comment]
"I want to work on this issue under JWOC"

**Comment by codervivek5** [comment]
@Beingfarazkhan  there is any update? Are you working on it?

**Comment by Code1705** [comment]
I want to work on this issue , please assign me this under JWOC

**Comment by Beingfarazkhan** [comment]
@codervivek5 i was working on it but its not showing under my assigned now

**Comment by Beingfarazkhan** [comment]
@codervivek5 Yes i am working on it , the designing is complete , now the implementation will be done in an hour. then i will send you a pr to review

**Comment by codervivek5** [comment]
ok 
But if possible then send screenshot so that I can review

**Comment by Beingfarazkhan** [comment]
@codervivek5 one more thing , i dont know if you have noticed but when i checked the code for other pages it might cause some bugs while switching through the pages.

**Comment by codervivek5** [comment]
yeah I notice many thing. there are a lot of bugs and to fix them. but if project is ready then we will work on bugs

**Comment by codervivek5** [comment]
so I am telling you that if you create any UI or any developed about us page then share screenshot then I will review it

**Comment by Beingfarazkhan** [comment]
This was the rough layout i provided
![Screenshot (1278)](https://user-images.githubusercontent.com/97282347/219488255-895b9896-d7e7-441e-b69f-624ba4ff01a7.png)
![Screenshot (1279)](https://user-images.githubusercontent.com/97282347/219488266-06b05281-f77a-4896-ba42-73edcc399d6b.png)

![Screenshot (1280)](https://user-images.githubusercontent.com/97282347/219488269-063d52b7-e472-4d99-beb4-d15ea18943e4.png)

**Comment by Beingfarazkhan** [comment]
@codervivek5 Sir if the pr is merged then please close this issue. Thank you!

**Comment by codervivek5** [comment]
Make your PR. I will merge it then close

**Comment by codervivek5** [comment]
This is perfect 👍
